jenkins/bootstrap_test.py

```diff
@@ -1133,6 +1133,14 @@ def testAllJobsHaveErrExit(self):
                      expected, lines,
                      '%s not found in %s' % (expected, job_path))
 
+    def testNoBadVarsInJobs(self):
+        """Searches for jobs that contain ${{VAR}}"""
+        for job, job_path in self.jobs:
+            with open(job_path) as fp:
+                script = fp.read()
+            bad_vars = re.findall("(\${{.+}})", script)
```

Comment:
changed `"` to `r'`.

the `\$` is necessary because `$` has special meaning for regular expressions.